frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.css

```diff
@@ -58,7 +53,8 @@
 }
 
 .TableInteractive .TableInteractive-header,
-.TableInteractive .TableInteractive-header .TableInteractive-cellWrapper {
+.TableInteractive .TableInteractive-header .TableInteractive-cellWrapper,
+.TableInteractive .TableInteractive-header .TableInteractive-cellWrapper:hover {
```

Comment:
There is a conflict in CSS selector specificity in emotion because there is an extra class for the root element. That's why I need to this extra `:hover` to override this.